acme_diags/run.py

```diff
@@ -188,8 +188,8 @@ def _remove_attrs_with_default_values(self, param):
 
     def _find_attrs_with_none_default_values(self, param):
         """
-        In the param, remove any parameters that
-        have their default value.
+        In the param, find any parameters that
+        have none default value.
```

Comment:
I'd say `default value of None` just to match Python's casing of `None`.

Reply:
Thanks for reviewing. It appears to be a misspelling. I updated the PR.